api/internal/init.js

```diff
@@ -267,25 +267,22 @@ class RuntimeXHRPostQueue extends ConcurrentQueue {
     const options = { responseType: 'arraybuffer' }
     const result = await ipc.request('post', { id }, options)
 
+    promise.resolve()
```

Comment:
would it make sense if will resolve on `!result.err` and reject (along with dispatching an event) otherwise?

Reply:
we don't want to reject - this is just me moving code around to be a bit cleaner!